src/interface/update.cpp

```diff
@@ -60,7 +60,7 @@ template <>
 TaskStatus FluxDivergence(MeshData<Real> *in_obj, MeshData<Real> *dudt_obj) {
   const IndexDomain interior = IndexDomain::interior;
 
-  std::vector<MetadataFlag> flags({Metadata::WithFluxes});
+  std::vector<MetadataFlag> flags({Metadata::WithFluxes, Metadata::Cell});
   const auto &vin = in_obj->PackVariablesAndFluxes(flags);
```

Comment:
should there be an equivalent functionality for face fields? I suppose it will necessarily look different

Reply:
Yeah, the natural version for face fields is to update them based on a finite-volume curl. The index ranges are different as well, so I think it needs to be a different task (or at least a separate kernel within this task).

Reply:
I would put off adding update functions for non-cell centered fields to a future PR, since they are just convenience functions and it might be useful to have a little experience using this machinery in downstream codes before including them.